config/load.go

```diff
@@ -3,11 +3,34 @@ package config
 import (
 	"encoding/json"
 	"io/ioutil"
+	"os"
 	"strings"
 
 	"gopkg.in/yaml.v2"
 )
 
+func loadEnvironmentVariable(p *Project) {
+	if strings.HasPrefix(p.Secret, "$") {
+		tempString := strings.TrimPrefix(p.Secret, "$")
+		tempEnvVar, doesItExist := os.LookupEnv(tempString)
```

Comment:
Let's rename doesItExist to `p` or `present`?

Reply:
okay, all done